great-idea/css/index.css

```diff
@@ -63,4 +63,82 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Your code starts here! */
+body {
+	width: 960px;
+	margin: 0 auto;
+	padding: 10px 20px;
+}
+.wrapper {
+	display: flex;
+	flex-direction: column;
+}
+header {
+	display: flex;
+	flex-direction: column;
+}
+.main-header {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	align-items: flex-start;
+	margin-bottom: 30px;
+}
+nav a {
+	text-decoration: none;
+	color: grey;
+	padding: 20px;
+}
+.subheader {
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+	align-items:flex-start;
+	align-content: space-between;
```

Comment:
You probably are trying to leave some spaces between these two elements in the **subheader** div. To make it work you need just;

**justify-content: space-between**
and
**align-items: center**

please fix!